drift/core/psestimation.py

```diff
@@ -625,16 +627,55 @@ def q_estimator(self, mi, vec1, vec2=None, noise=False):
         x1 = np.dot(evecs.T.conj(), x0)
 
         # Project back into sky basis
-        x2 = self.kltrans.beamtransfer.project_vector_svd_to_sky(mi, x1, conj=True)
+        x2 = self.kltrans.beamtransfer.project_vector_svd_to_sky(
+            mi, x1, temponly=True, conj=True
```

Comment:
Why the change to `temponly=True` here and below?

Reply:
Yeah I am open to discuss this. For the power spectrum you only need the temperature part. But if you were to use this function 'project_vector_kl_to_sky' in a different context where you would want polarization might be better to keep it general..

Reply:
Never mind, I agree with `temponly=True`